docs/docs/operations/logging.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Logging the request body pattern for a request
+nav_order: 5
+parent: Operations
+---
+
+# Logging the request body pattern for a request
+
+If you want to log the request body in the `customErrorHandler` middleware, unfortunately the request body has been consumed in the `customErrorHandler` middleware and can't be read again. To log the request body, you can use one middleware to buffer the request body before it's consumed.
```

Comment:
I think we can be more general than this. How about
```suggestion
If you want to log the request body of incoming requests, you will need to buffer the body before it reaches the gateway. To log the request body, you can use a middleware `http.Handler` to buffer the request body before it's consumed.
```

Reply:
I will update the doc